cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/DefaultSecureStoreServiceTest.java

```diff
@@ -76,8 +81,12 @@
 
   @BeforeClass
   public static void setup() throws Exception {
-    CConfiguration cConf = createCConf();
-    final Injector injector = AppFabricTestHelper.getInjector(cConf);
+    SConfiguration sConf = SConfiguration.create();
+    sConf.set(Constants.Security.Store.FILE_PASSWORD, "secret");
+    final Injector injector = Guice.createInjector(new AppFabricTestModule(createCConf(), sConf));
+    injector.getInstance(TransactionManager.class).startAndWait();
+    injector.getInstance(DatasetOpExecutor.class).startAndWait();
+    injector.getInstance(DatasetService.class).startAndWait();
```

Comment:
I think it may be a good idea to wait for dataset service to become discoverable. look for other tests that do that. MetadataAdminAuthorizationTest, etc.